app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -78,3 +103,74 @@ private fun DeleteAccountDialog(
 		text = { Text(stringResource(R.string.DeleteAccountConfirmationDialogDescription)) },
 	)
 }
+
+@Composable
+private fun ProfileSegment(
+	state: ViewModelState<Profile>,
+) {
+	if (state.isLoaded()) {
+		UserAvatar(state.data().photo)
+		Divider(
+			modifier = Modifier
+				.fillMaxWidth()
+				.padding(all = 14.dp),
+			thickness = 1.dp,
+		)
+		Text(
+			text = stringResource(id = R.string.UserName),
```

Comment:
Inconsistency: Here you are using named parameters `text = ...` but in the next item you are passing text without using named parameters. Also the `modifier` should be the first parameter.

Reply:
Ok, I see I forgot to add `text = ...`, and I added a modifier as a second parameter because of it 
![image](https://github.com/intive/Picover/assets/57491794/db1a76b6-3119-4010-9698-036226a5263d)
but I am not attached to it so I can move it to the top. I remember we probably discuss it lately and I probably didn't find any official guide on where it should be. 

I looked at the whole project and I found only one place where the modifier is in the second position 
![image](https://github.com/intive/Picover/assets/57491794/8310c5be-64eb-4604-baf1-9c145231b675)
so to make my changes more consistent I will move the modifier to the top, also I don't remember if we discuss it at the guild meeting or if it wasn't necessary😄.

Reply:
Done 😄